crt_portal/cts_forms/templates/forms/snippets/sortable_table_heading.html

```diff
@@ -2,18 +2,18 @@
 
 {% load static %}
 
-<th scope="col" class="{% if sort_url %}sort-cell{% endif %}" {% if nowrap %}nowrap{% endif %}>
+<th aria-label="{{heading}}" scope="col" class="{% if sort_url %}sort-cell{% endif %}" {% if nowrap %}nowrap{% endif %}>
   {% if sort_url %}
     <a
       id="{{id}}"
       href="{{sort_url}}{{filter_state}}"
       class="sort-link display-block {% if is_descending %}sort-up{% else %}sort-down{% endif %}"
       aria-label="Sort complaints by {{ heading }} {% if is_descending %}descending{% else %}ascending{% endif %}"
     >
-      <span>{{ heading }}</span>
+      <span aria-hidden="true">{{ heading }}</span>
```

Comment:
Ok, so there's some weird stuff that happens here, really only for the headings without `<a>` tags as children. If it's just an aria-hidden span element, the heading never receives focus when tabbing through. Which is really problematic for keyboard users. Even when using a screen reader the element still doesn't receive visual focus. I would suggest wrapping the aria hidden span elements in something like a tab-indexed `<div>` element and see if that adds back keyboard focus.

Also, NVDA doesn't read these in Edge. It skips over Contact Details and Incident City/State headings.

Reply:
Nice, good catch - adding tabindex seems to have added back tab/voiceover focus for the headers and cells